reascripts/ReaSpeech/source/ReaSpeechWorker.lua

```diff
@@ -198,9 +216,24 @@ function ReaSpeechWorker:handle_job_status(active_job, response)
   end
 
   if response.job_status == 'SUCCESS' then
-    local transcript_url_path = response.job_result.url_path
+    local job_result = response.job_result
+    if job_result.result then
+      self:handle_response(active_job, job_result.result)
+      self.active_job = nil
+      return false
+    end
+
     response._job = active_job.job
-    active_job.transcript_request = ReaSpeechAPI:fetch_large(transcript_url_path)
+
+    if response.job_result.url_path then
+      table.insert(active_job.requests, ReaSpeechAPI:fetch_large(response.job_result.url_path))
+    elseif response.job_result.url_paths then
+      for _, url_path in pairs(response.job_result.url_paths) do
+        table.insert(active_job.requests, function()
+           ReaSpeechAPI:fetch_large(url_path)
```

Comment:
Do we no longer need the return value of fetch_large?

Reply:
good catch, yeah! i'm not sure how i clobbered this line 🤔